modules/runtime-common/src/test/scala/com/snowplowanalytics/snowplow/runtime/ConfigParserSpec.scala

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Snowplow Community License Version 1.0,
+ * and you may not use this file except in compliance with the Snowplow Community License Version 1.0.
+ * You may obtain a copy of the Snowplow Community License Version 1.0 at https://docs.snowplow.io/community-license-1.0
+ */
+package com.snowplowanalytics.snowplow.runtime
+
+import java.nio.file.Paths
+import io.circe.literal._
+import io.circe.Decoder
+import io.circe.generic.semiauto._
+import cats.effect.IO
+import cats.effect.testing.specs2.CatsEffect
+import org.specs2.Specification
+import com.snowplowanalytics.iglu.client.resolver.Resolver
+
+class ConfigParserSpec extends Specification with CatsEffect {
+  import ConfigParserSpec._
+
+  def is = s2"""
+  ConfigParser should
```

Comment:
Not sure if we use it somewhere, but should we add also case for [namespaced](https://github.com/snowplow-incubator/common-streams/blob/76ee3de57072c24d719b42071252925f95910d24/modules/runtime-common/src/main/scala/com/snowplowanalytics/snowplow/runtime/ConfigParser.scala#L84) configuration?